ship/src/main/scala/ch/epfl/bluebrain/nexus/ship/config/S3Config.scala

```diff
@@ -0,0 +1,14 @@
+package ch.epfl.bluebrain.nexus.ship.config
+
+import pureconfig.ConfigReader
+import pureconfig.generic.semiauto.deriveReader
+
+final case class S3Config(endpoint: String, importBucket: String)
```

Comment:
There may be a reader for URI so you could directly use it here
For the bucket, you may have one too but adding another library for refined is a bit overkill for now.
An alternative could be to use `Label` and add a way to translate it to a `NonEmptyString`

Reply:
I made the change for URI. For the bucket I don't really see the value of using the NonEmptyString here is, vs. the change I made which encapsulates the refinement within the implementation. And even if we went for using the BucketName in the config, we still need to handle the FileKey somewhere else... so the way it is now at least covers both cases at once. And it's not like we are going to start running the tool and specifying an empty string for the bucket